goship.go

```diff
@@ -517,7 +528,11 @@ func DeployHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	p := r.FormValue("project")
 	env := r.FormValue("environment")
-	user := r.FormValue("user")
+	u, err := getUser(r)
+	if err != nil {
+		log.Fatal("Failed to get a user while deploying!")
```

Comment:
I believe `log.Fatal` will kill the server for everyone, it's better to use `log.Println` and then return an error code.